tensorboard/webapp/plugins/npmi/views/inactive/inactive_container.ts

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component} from '@angular/core';
+import {Store} from '@ngrx/store';
+
+/** @typehack */ import * as _typeHackRxjs from 'rxjs';
+
+@Component({
+  selector: 'npmi-inactive',
+  template: `
+    <inactive-component></inactive-component>
+  `,
+})
+export class InactiveContainer {}
```

Comment:
In case you do not require access to the store (or dispatch actions to the store), you can omit the container. Unless there are plans to add content here that requires information from the store, omit the container?

Reply:
Done.